### PR TITLE
[SMA] Correcting CL tests

### PR DESCRIPTION
**Pull request by AlejandroCornejo**
https://github.com/KratosMultiphysics/Kratos/pull/7771

### CODE REVIEW

**Comment by philbucher** [comment]
there are no changes :sweat_smile:

**Comment by AlejandroCornejo** [comment]
Yes, it seems that the changes have been undone WTF. let's see if the tests run

**Comment by AlejandroCornejo** [comment]
> 
> 
> Yes, it seems that the changes have been undone WTF. let's see if the tests run

in my PC are running ok

**Comment by philbucher** [comment]
the failing tests are in the nightly suite, hence they will not show up here I think

**Comment by AlejandroCornejo** [comment]
> 
> 
> the failing tests are in the nightly suite, hence they will not show up here I think

How can i reproduce them?

**Comment by marcnunezc** [comment]
> > the failing tests are in the nightly suite, hence they will not show up here I think
> 
> How can i reproduce them?

Running this file fails in the same test as the CI for me:

https://github.com/KratosMultiphysics/Kratos/blob/master/applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py

**Comment by philbucher** [comment]
@AlejandroCornejo now you can see the error:
~~~
======================================================================
FAIL: test_execution (structural_mechanics_test_factory.SmallDeformationPlasticityTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\a\Kratos\Kratos\bin\Custom\applications\StructuralMechanicsApplication\tests\structural_mechanics_test_factory.py", line 72, in test_execution
    self.test.RunSolutionLoop()
  File "D:\a\Kratos\Kratos\bin\Custom\KratosMultiphysics\analysis_stage.py", line 69, in RunSolutionLoop
    self.FinalizeSolutionStep()
  File "D:\a\Kratos\Kratos\bin\Custom\KratosMultiphysics\analysis_stage.py", line 151, in FinalizeSolutionStep
    process.ExecuteFinalizeSolutionStep()
  File "D:\a\Kratos\Kratos\bin\Custom\KratosMultiphysics\from_json_check_result_process.py", line 55, in ExecuteFinalizeSolutionStep
    self.assertTrue(self.process.IsCorrectResult(), "Results do not coincide with the JSON reference\n" + self.process.GetErrorMessage())
AssertionError: False is not true : Results do not coincide with the JSON reference
Error checking for variable UNIAXIAL_STRESS in GP 0, results:
Element 1 2980492!=3000002; rel_tol=1e-06, abs_tol=0.001
Error checking for variable UNIAXIAL_STRESS in GP 1, results:
Element 1 2980492!=3000002; rel_tol=1e-06, abs_tol=0.001
Error checking for variable UNIAXIAL_STRESS in GP 2, results:
Element 1 2980492!=3000002; rel_tol=1e-06, abs_tol=0.001
Error checking for variable UNIAXIAL_STRESS in GP 3, results:
Element 1 2980492!=3000002; rel_tol=1e-06, abs_tol=0.001
Error checking for variable UNIAXIAL_STRESS in GP 4, results:
Element 1 2980492!=3000002; rel_tol=1e-06, abs_tol=0.001
Error checking for variable UNIAXIAL_STRESS in GP 5, results:
Element 1 2980492!=3000002; rel_tol=1e-06, abs_tol=0.001
Error checking for variable UNIAXIAL_STRESS in GP 6, results:
Element 1 2980492!=3000002; rel_tol=1e-06, abs_tol=0.001
Error checking for variable UNIAXIAL_STRESS in GP 7, results:
Element 1 2980492!=3000002; rel_tol=1e-06, abs_tol=0.001
~~~

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -298,4 +298,4 @@ jobs:
       run: |
         set PYTHONPATH=%PYTHONPATH%;%GITHUB_WORKSPACE%/bin/%KRATOS_BUILD_TYPE%
         set PATH=%PATH%;%GITHUB_WORKSPACE%/bin/%KRATOS_BUILD_TYPE%/libs
-        python kratos/python_scripts/run_tests.py -l small -c python
+        python kratos/python_scripts/run_tests.py -l nightly -c python
```
Comment (AlejandroCornejo) [inline .github/workflows/ci.yml]: This is what I wanted to know! thx @philbucher

**Comment by AlejandroCornejo** [comment]
Let's see if now is okkey

File: .github/workflows/ci.yml (hunk 1 above)
Comment (philbucher) [reply .github/workflows/ci.yml]: it is a hack, but unfortunately as of now there is no better way to do this :/
I hope to soon have time to improve it!

**Comment by philbucher** [comment]
now it works!

**Review by philbucher — APPROVED** [review]

**Comment by AlejandroCornejo** [comment]
Nice! sorry for the inconvenience...

**Comment by philbucher** [comment]
can you please use a squash merge to not have the ci-commits in the history? Thx

**Pull request merged**